packages/inspection-capture-web/src/PhotoCapture/PhotoCaptureHUD/PhotoCaptureHUD.tsx

```diff
@@ -96,12 +102,23 @@ export function PhotoCaptureHUD({
   const { t } = useTranslation();
   const [showCloseModal, setShowCloseModal] = useState(false);
   const style = usePhotoCaptureHUDStyle();
+  const { trackEvent } = useAnalytics();
 
   const handleCloseConfirm = () => {
     setShowCloseModal(false);
+    trackEvent('close confirmed', {
```

Comment:
Just a quick note : is this the formatting used and recommended by posthog for event names? All lowercase names?

Also, this event should be called `capture closed` imo

Reply:
The only recommendation from Posthog is to use '[object][verb]' format for event names.